code/FinanceManager.Components/Features/FinancialAccounts/Components/Shared/AccountHistoryToolbar.razor.css

```
@@ -5,6 +5,27 @@
+    --fm-toolbar-control-height: 32px;
+}
+
+.fm-tx-toolbar ::deep .fm-toolbar-search-button {
+    width: var(--fm-toolbar-control-height);
+    height: var(--fm-toolbar-control-height);
+    border: 1px solid var(--mud-palette-lines-inputs);
+    border-radius: var(--mud-default-borderradius);
+    box-sizing: border-box;
+}
+
+.fm-tx-toolbar ::deep .mud-toggle-item,
```

Stylelint (17.14.0)

[error] Unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)